Gemfile.lock

```diff
@@ -244,6 +244,9 @@ GEM
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       sprockets (>= 2.8, < 4.0)
+    sqlite3 (1.3.10)
```

Comment:
I give you a small challenge: try to fix the gemspec such that it doesn’t need it, then PR.

Hint: it’s a development dependency (I think…)